core/trino-spi/src/main/java/io/trino/spi/expression/Function.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.expression;
+
+import io.trino.spi.type.Type;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class Function
+        extends ConnectorExpression
+{
+    public static final String LIKE_FUNCTION_NAME = "like-function";
```

Comment:
The ConnectorExpression abstraction shouldn't be aware of any function names, so move this constant somewhere else, ideally, where it's being used.

Reply:
Moved into `ConnectorExpressionTranslator`